app/views/subscriptions/new.html.erb

```diff
@@ -10,12 +10,24 @@
 
 <%= render 'govuk_component/breadcrumbs', breadcrumbs: @breadcrumbs %>
 
-<%= render "components/back_link", {
+<%= render "govuk_publishing_components/components/back_link", {
   href: @back_url,
 } %>
 
 <div class='grid-row'>
   <div class='column-two-thirds'>
+    <% if flash[:error] %>
+      <%= render "govuk_publishing_components/components/error_summary", {
+        title: "This isn't a valid email address",
```

Comment:
Minor: Text could use smart quotes for better readability (http://smartquotesforsmartpeople.com/)

Reply:
Fixed